OBAApplicationDelegate.m

```diff
@@ -63,7 +68,14 @@ - (id)init {
                                                                                 [self writeSetRegionAutomatically:YES];
                                                                                 [self.regionHelper updateNearestRegion];
                                                                             }];
-
+        self.recentStopsObserver = [[NSNotificationCenter defaultCenter] addObserverForName:OBAMostRecentStopsChangedNotification
```

Comment:
Much obliged. You're completely right that this will never be invoked as-is, but in the event someone someday refactors this code and moves your observer block out of app delegate, things may change. Having the `-dealloc` code sitting there will be a good reminder to them of what they have to do to prevent crashes :)